Emergence/Assets.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -3,6 +3,7 @@
     {
       "orientation" : "landscape",
       "idiom" : "tv",
+      "filename" : "launch.png",
       "extent" : "full-screen",
       "minimum-system-version" : "9.0",
       "scale" : "1x"
```

Comment:
added all the initial logo stuff, it's got a nice preview

![screen shot 2015-09-29 at 17 22 48](https://cloud.githubusercontent.com/assets/49038/10170819/bc726d42-66ce-11e5-9ca3-762a50089f8a.png)